src/components/QuranReader/ReadingView/ReadingViewSkeleton.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import range from 'lodash/range';
+import { shallowEqual, useSelector } from 'react-redux';
+
+import styles from './ReadingPreviewSkeleton.module.scss';
+
+import Skeleton from 'src/components/dls/Skeleton/Skeleton';
+import { selectQuranReaderStyles } from 'src/redux/slices/QuranReader/styles';
+import QuranReaderStyles from 'src/redux/types/QuranReaderStyles';
+
+const ReadingPreviewSkeleton = () => {
+  const numberOfLines = 15;
```

Comment:
will need to do follow up PR related to how many skeleton we want to render

Reply:
The skeleton always shows a full page. Number of lines should read from redux